lib/levelup.js

```diff
@@ -53,8 +54,23 @@ function LevelUP (db, options, callback) {
 
   this.options = getOptions(options)
   this._db = db
-  this.db = new DeferredLevelDOWN(db)
+  this.db = db.supports && db.supports.deferredOpen ? db : new DeferredLevelDOWN(db)
```

Comment:
Should also add this logic to `levelup#close()`, but this is just a proof of concept, I'll revert dc5cdb779592dbeca81b66d2a5305697f4ee05a1 after review (I hope you can see where I'm going with this 😃)

Reply:
Hmm, don't you mean the opposite of this? I.e. `db.supports && db.supports.deferredOpen ? new DeferredLevelDOWN(db) : db`

Reply:
Oh nvm. I get it now :)

Reply:
E.g. if the db itself supports deferred open, there's no need in wrapping it with `DeferredLevelDOWN()`.

Reply:
✔️